### PR TITLE
#1-4 Task. Dmitry Astapchik

### PR DESCRIPTION
**Pull request by DmitryAstapchik**
https://travis-ci.org/github/DmitryAstapchik/js-assignments/builds/704780808

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -201,7 +201,39 @@ function extractEmails(str) {
...
+        }
+        else if (i !== 0 && i !== height - 1) {
+            rect += '│\n';
+        }
+        else {
+            rect += '┘\n';
+        }
+    }
+    return rect;
 }
```
Comment (FedorovD) [inline task/01-strings-tasks.js:237]: `var` is deprecated. It has no sense to use for browsers code even currently.

File: task/01-strings-tasks.js [hunk 2]
```diff
@@ -221,7 +253,17 @@ function getRectangleString(width, height) {
...
+        encoded += encode(str[i]);
+    }
+    return encoded;
+
+    function encode(c) {
+        var input = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+        var output = 'NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm';
+        return input.indexOf(c) === -1 ? c : output[input.indexOf(c)];
+    }
 }
```
Comment (FedorovD) [inline task/01-strings-tasks.js:267]: `var` is deprecated. It has no sense to use for browsers code even currently.

**Comment by FedorovD** [comment]
Good, but in the future use let/const instead of var 
**Closing the PR as accepted! Thanks!**

File: task/01-strings-tasks.js (hunk 1 above)
Comment (DmitryAstapchik) [reply task/01-strings-tasks.js:237]: ok i got it

File: task/01-strings-tasks.js (hunk 2 above)
Comment (DmitryAstapchik) [reply task/01-strings-tasks.js:267]: ok i got it

**Pull request closed without merging**